web/package-lock.json

```diff
@@ -2312,7 +2312,7 @@
     },
     "node_modules/svelte-utils": {
       "version": "0.0.1",
-      "resolved": "git+ssh://git@github.com/a-b-street/svelte-utils.git#ebe8244f143467088040aab01c86a19f13809f19",
+      "resolved": "git+ssh://git@github.com/a-b-street/svelte-utils.git#e9d4cefc8875bfe1ebfae7fa47de7574ec1c1c4f",
```

Comment:
Ah, this should have been my cue to run npm ci.

Reply:
I miss `cargo` and not worrying about this kind of thing :(